strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-raw-layout-provider/src/test/java/org/carlspring/strongbox/config/RawLayoutProviderTestConfig.java

```diff
@@ -8,6 +8,7 @@
  */
 @Configuration
 @Import({ RawLayoutProviderConfig.class,
+          EventsConfig.class,
```

Comment:
... or not? :)

Reply:
I can't say for sure is it cleanup or not :)
but you should know that we have a real problem with Spring configuration classes and @CompontntScan. The problem is that all our modules share same package names like `org.carlspring.strongbox.services` or `org.carlspring.strongbox.config` which cause and this ambiguous behavior of @CompontntScan annotation. We can't say for sure what configuration will fetch beans from such packages and how to manage bean dependencies correctly.

Reply:
I need to say I have the same problem from time to time.

Reply:
@sbespalov : Thanks for the thorough explanation! We'll have to look into this at some point, as @fuss86 has also had to fix a lot of such context issues in the past. Could either of you add a follow-up task for this in youtrack (with a proper explanation)...? Thanks! :)

Reply:
ok

Reply:
SB-1189